internal/pkg/levee/levee.go

```diff
@@ -48,6 +51,15 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+	if conf.UseEAR() {
+		return runEAR(pass) // Use the EAR-pointer based taint analysis
+	} else {
+		return runPropagation(pass) // Use the propagation based taint analysis
+	}
+}
+
+func runPropagation(pass *analysis.Pass) (interface{}, error) {
+	conf, _ := config.ReadConfig()
```

Comment:
Prefer passing the config as an argument, instead of reading it again and ignoring the error.